src/globus_sdk/experimental/globus_app/authorizer_factory.py

```diff
@@ -141,9 +153,10 @@ def _make_authorizer(self, resource_server: str) -> RefreshTokenAuthorizer:
         :param resource_server: The resource server the authorizer will produce
             authentication for
         """
-        token_data = self._get_token_data_or_error(resource_server)
+        token_data = self.token_storage.get_token_data(resource_server)
         if token_data.refresh_token is None:
-            raise MissingTokensError(f"No refresh_token for {resource_server}")
+            msg = f"No refresh_token for {resource_server}"
+            raise MissingTokensError(msg, resource_server=resource_server)
```

Comment:
The docstring isn't implicated in this change so I can't highlight it, but for consistency with other changes, could the narrative "Raises blah blah blah" convert to autodoc-compatible markup, like `:raises: :exc:...`?

Reply:
Sure, updated in new rev